lib/services/components.js

```diff
@@ -469,8 +473,12 @@ function getStyles(name, slug) {
 function getScripts(name, slug) {
   const site = siteService.sites()[slug],
     assetDir = site && site.assetDir || mediaDir,
-    assetDirLen = assetDir.length,
-    assetPath = site && site.assetPath || '';
+    assetDirLen = assetDir.length;
+  let assetPath = site && site.assetPath || '';
+
+  if (site && site.path && site.assetPath !== site.path) {
+    assetPath = site.path;
+  }
```

Comment:
I think I understand.

For number two, why is it good to have an asset path like that?  Wouldn't that cause `http://vulture.com/vulture/css/simple-header.css` ?

There is this code in sites.js:
```
    siteConfig.slug = site;
    siteConfig.path = normalizePath(siteConfig.path);
    siteConfig.dir = dir;
    siteConfig.assetPath = normalizePath(siteConfig.assetPath);
    siteConfig.assetDir = normalizeDirectory(siteConfig.assetDir || 'public');
    siteConfig.prefix = siteConfig.path && siteConfig.path.length > 1 ? siteConfig.host + siteConfig.path : siteConfig.host;
```

which I think means the configs were meant to be something like this for vulture:
```
proto: (defaults to http)
port: (defaults to env.PORT)
slug: (defaults to name of the folder)
host: vulture.com
path: (defaults to /)
assetPath: (defaults to /)
assetDir: (defaults to public)
```

At least, that was the intention.